config/toml.go

```diff
@@ -372,6 +361,18 @@ func ResetTestRootWithChainID(testName string, chainID string) *Config {
 	return config
 }
 
+func mustCreateTestRootDir(testName, chainID string) string {
+	homeDir := os.ExpandEnv("$HOME/.tendermint_test")
+	if err := os.MkdirAll(homeDir, 0777); err != nil {
```

Comment:
Shouldn't this be `0700`? At least later we check for `cmn.EnsureDir(rootDir, 0700)`. 
```suggestion
	if err := os.MkdirAll(homeDir, 0700); err != nil {
```

Reply:
Depends. On UNIX this will be modifed by the user's umask anyway, it will rarely ends up as 0777 really. Furthermore, why do we negate user's group access to a test directory? Is there any particular security concern? Not a big deal though, happy to amend it!